src/store/api-call.js

```diff
@@ -0,0 +1,53 @@
+import { BASEURL } from "../assets/apis/config";
+
+const getDataRequest = async (url) => {
+  const response = await fetch(url);
+  const data = await response.json();
```

Comment:
Why you create local variable for data? you make it inline like response

Reply:
I could return response.json I just made data to look cleaner. plus it is a common convention to store it in a local variable when not using .then